GNUmakefile

```diff
@@ -22,7 +22,7 @@ test: fmtcheck
 	go test -i $(TEST) || exit 1
 	echo $(TEST) | \
 		xargs -t -n4 go test $(TESTARGS) -timeout=30s -parallel=4
-	DD_API_KEY=fake DD_APP_KEY=fake RECORD=false TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=120s
+	DD_API_KEY=fake DD_APP_KEY=fake RECORD=false TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=10m
```

Comment:
10 minutes? It feels like quite a long time.

Reply:
I can try with 5m but i don't think i can go lower than that now

Reply:
6m seems to be the minimum i can have here